plugins/cost.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cost
+spec:
+  version: v0.1.4
+  shortDescription: View cluster cost information
+  description: |
```

Comment:
Looking at https://github.com/kubecost/kubectl-cost#installation I get the impression that this plugin requires a cluster-side daemon. You should mention this in the `caveats` section.

Reply:
Addressed in the latest commit.